### ISSUE TITLE
[1817] System should force all new corps to buy tokens at the end of the stock round (if not enough money, should liquidate)

### ISSUE THREAD

**Issue by bridger15** [tobymao/18xx#3735]
So here's an intresting issue: https://boardgamegeek.com/thread/2597058/drowning-short-defenders-loan-fees-using-new-start

TL;DR:
Player A shorts company belonging to player B
Player B takes loans to redeem shares to defend against shorts
Player A starts new (cheap) 10 share company and immediately takes out 10 loans (significantly increasing interest on Player B's company)
(Repeat as necessary to drive up interest rate)
Player A purposefully fails to purchase tokens for his new companies
All new companies immediately go to liquidation, bypassing the need to pay interest
(in the future M&A Phase) New companies still have all the money from the loans, so they pay them off with Player A only losing his (small) initial president cert investment.

Here are the relevant rules: 
"New companies receive the initial station marker placed on the map for free. **5-share companies must purchase 1 additional station marker for $50, and 10-share companies _must_ purchase 3 additional station markers for $150."** 

"Payment for these station markers must be made prior to the end of the stock round. Paying for these station markers is not an action. Companies may take loans to pay for the initial station markers. Taking loans must be a company action. **If the stock round ends without the company purchasing the required station markers, the company is placed in liquidation.** After the IPO is concluded, play proceeds to the left of the player that started the IPO auction, regardless of who won the auction."

That 2nd part in bold is, I think, referring to a situation where a company didn't have enough money to satisfy the previously quoted section.  It's not designed to allow a company to CHOOSE not to buy tokens if it has money.  The first bold section certainly seems to indicate that if the company does have the money, it MUST buy the tokens.

**Comment by jenf** [comment]
As discussed should be automatic if corp has enough cash either when setup or after taking loan. The warning still needs to exist for the case that corp needs to take loans

**Comment by jenf** [comment]
Will need a migration of 1817, 1817NA, 1817WO and 1877
